Alexmnet news

AlexMNet opened the pull request.

Commit:
News page styles

Adjusted styles of the news page to closer resemble webflow design
while retaining the essence of the current design.

Headline and summary font is bigger.
Photos are bigger from 200px to 300px

Light gray border added to each list item to make it easier to make a distinction between each li

Commit:
Merge branch 'main' into alexmnet-news
Update branch with main

Commit:
testing browser refresh

mckoss commented on themes/ra-theme/assets/css/header.scss:
```diff
@@ -35,7 +35,7 @@ header {
     border: 0;
     background-image: url("/images/icon-hamburger.svg");
-    background-color: transparent;
+    background-color: 'transparent';
```
no quotes

mckoss commented on themes/ra-theme/assets/css/header.scss:
```diff
@@ -53,7 +53,7 @@ header {
         z-index: 90;
         inset: 0 0 0 30%;
-        background-color: hsla(0, 0%, 100%, 0.5);
+        background-color: 'hsla(0, 0%, 100%, 0.5)';
```
no quotes

mckoss commented on themes/ra-theme/assets/css/summary.scss:
```diff
@@ -3,13 +3,17 @@ $summary-break: 600px;
   margin: 1rem auto;
   padding: 0;
-  max-width: 800px;
+  max-width: 990px;
```
Please add as a breakpoint in the breakpoint.scss file.

mckoss reviewed (changes requested):
A few changes to make.

Commit:
PR Changes

Remove quotes from header sass file.
Remove center alignment for images
Add breakpoint max-sumamry-width to breakpoints sass file

AlexMNet commented:
- Removed the quotes from the header sass file
- Removed center alignment on images
- Added max-summary-width of 990px to breakpoints

mckoss reviewed (approved):
Looks great, Alex.  "LGTM"
